docs/content/how-to/disaster-recovery/backup-and-restore-oadp.md

```diff
@@ -145,6 +152,111 @@ Below are some samples of DPA configurations for the mentioned platforms
             - openshift
             - aws
             - csi
+          customPlugins:
+            - name: hypershift-oadp-plugin
+              image: quay.io/redhat-user-workloads/ocp-art-tenant/oadp-hypershift-oadp-plugin-main:main
+          resourceTimeout: 2h
```

Comment:
Why 2h?

Reply:
I don't think it will take so long, but I was thinking in the worst scenario 3 full Etcds 24 gb.